kymatio/frontend/tensorflow_frontend.py

```diff
@@ -0,0 +1,20 @@
+import tensorflow as tf
+from packaging import version
+
+assert version.parse(tf.__version__)>=version.parse("2.0.0a0"), 'Current TensorFlow version is '+str(tf.__version__)+\
+                                                                ' . Please upgrade TensorFlow to 2.0.0a0 at least.'
+
+from abc import ABCMeta, abstractmethod
+
+class ScatteringTensorflow(tf.Module, metaclass=ABCMeta):
```

Comment:
`ScatteringTensorflow` or `ScatteringTensorFlow`?

Reply:
@edouardoyallon

Reply:
TensorFlow I guess... I'm not sure what is the standard way of doing it.

Reply:
It's normally written TensorFlow, so I suggest we stick to that.

Reply:
Alright.

Reply:
How important is it that that this be written TensorFlow? 

https://github.com/kymatio/kymatio/blob/kymatio-dev-frontend/kymatio/scattering1d/frontend/__init__.py We get errors thrown over here if we rename to TensorFlow, as `frontend  = "tensorflow"`, `frontend.capitalize() = "Tensorflow"`, and if we had a statement which renames `frontend = "tensorflow"` to `frontend = "tensorFlow"`, then `frontend.capitalize = "Tensorflow"`.

One thing we could do is change frontend name to have proper capitalization. So, in tensorflow frontend, `frontend = "TensorFlow"`, and in numpy frontend, `frontend = "NumPy"`.